script/tool/lib/src/version_check_command.dart

```diff
@@ -175,6 +184,14 @@ class VersionCheckCommand extends PluginCommand {
     }
     // Remove all leading mark down syntax from the version line.
     final String versionString = firstLineWithText.split(' ').last;
+
+    // Skip validation for the special NEXT version that's used to accumulate
+    // changes that don't warrant publishing on their own.
+    if (versionString == 'NEXT') {
```

Comment:
We also need to make sure that the version in pubspec equals to the next version after `NEXT` in CHANGELOG. 

This is to prevent someone updated the version in pubspec but forgot to change the "NEXT"

Reply:
If we only cares about presubmit, which looks like so in the current code (base-sha not configurable), we can easily do
```
if (versionString == 'NEXT' && pubspecVersion not changed) 
```

Reply:
Good call, added validation for that and added a test.